src/lineage/entrypoint.py

```diff
@@ -412,6 +422,12 @@ def main() -> None:
                 repo, lineage_id, local_branch
             )
             logging.info("Pull request branch is new: %s", branch_is_new)
+            # Check if a PR exists if the branch already exists
+            if not branch_is_new and not has_existing_pr(repo, pr_branch_name):
+                core.warning(
+                    "There is no Pull Request for the existing Lineage branch.",
```

Comment:
Remove unnecessary capitalization:
```suggestion
                    "There is no pull request for the existing lineage branch.",
```

Reply:
I was treating "lineage" as a proper noun in this case as I was implying ownership of the branch by this project. This project's naming is another hot mess because we talk about a project's lineage and how Lineage pushes upstream changes.

Reply:
In either case, "pull request" should not be capitalized.  I'm fine with Lineage being capitalized.

Reply:
`Lineage` is fine then.  My initial qualm was with "pull request", but then I flew too close to the sun.

Reply:
I submit for your consideration b043ad3.

Reply:
LGTM - thanks!